unit/generate_unit.go

```diff
@@ -480,6 +481,23 @@ var Units = []Unit{
 		},
 		ErForm: "Pressurer",
 	},
+	{
+		Name:        "Torque",
+		Receiver:    "t",
+		PrintString: "N m",
+		Suffix:      "newtonmeter",
+		Singular:    "Newtonmeter",
+		TypeComment: "Torque represents a torque in Newton meters",
+		Bug: `Torque is measured in N·m, which is dimensionally identical to energy.
```

Comment:
I still think this is a bit overblown, but there's no harm in having a longer comment on the type warning people about use. This is not a bug though.

Reply:
It's not a bug in the code here, it is a bug in the system. See https://golang.org/pkg/encoding/xml/#pkg-note-BUG which is also not a bug in the same way this isn't.